internal/webapp/views/pages/lobby_page.templ

```diff
@@ -1,16 +1,17 @@
-package views
-
-
-templ LobbyIndex(){
-@Template(){
-
-<div class="center">
-    <h1>سلام دنیا</h1>
-    <p class="hint" style="margin-bottom: 10px">در اینجا بزودی یک بازی قرار میگیرد!</p>
-    <button class="tg-button">یک دکمه پیشنهادی</button>
-</div>
-
-<script>
-</script>
-}
+package pages
+
+import "kingscomp/internal/webapp/views/layout"
+
+templ LobbyPage(){
+@layout.Base("Lobby"){
```

Comment:
Also there is no need to pass the title to the base layout, I will refactor this part myself using custom types.